volatility3/framework/plugins/linux/hidden_modules.py

```diff
@@ -29,9 +32,9 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
                 architectures=architectures.LINUX_ARCHS,
             ),
             requirements.VersionRequirement(
-                name="linux_utilities_modules",
-                component=linux_utilities_modules.Modules,
-                version=(3, 0, 0),
+                name="linux_utilities_modules_module_display_plugin",
```

Comment:
It would surely be better to define classmethod generator and run methods and then just proxy them at this point?  We're really not getting anything from inheritance other than complexity?

Reply:
Can `__init__` of each plugin just set `.run` and and `._generator` to the shared function?  and Where would that shared functons best live now? I was trying to avoid one plugin becoming the container (say lsmod) that the other plugins then need to import and depend on.

Reply:
Yeah, that should be ok, but you can just define it straight in the class I think?  So

```
class Blah:
  run = linux_utilities_modules.ModuleDisplayPlugin.run
```
?